Assets/Scripts/BlackHolePull.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class BlackHolePull : MonoBehaviour
+{
+    
+    public float gravitationalForce = 10;
+    
+    private List<Rigidbody> pulledBodies;
+    private SphereCollider pullCollider;
+    
+    private void Start()
+    {
+        pulledBodies = new List<Rigidbody>();
+        pullCollider = GetComponent<SphereCollider>();
+    }
+    
+    private void OnTriggerEnter(Collider other)
+    {
+        var pulledBody = other.gameObject.GetComponent<Rigidbody>();
+        if (pulledBody != null)
+        {
+            pulledBodies.Add(pulledBody);
+        }
+    }
+
+    private void OnTriggerExit(Collider other)
+    {
+        var pulledBody = other.gameObject.GetComponent<Rigidbody>();
+        RemovePulledBody(pulledBody);
+    }
+
+    private void FixedUpdate()
+    {
+        foreach (var pulledBody in pulledBodies)
+        {
+            var singularityPosition = transform.position;
+            var pulledBodyPosition = pulledBody.transform.position;
+            var direction = (singularityPosition - pulledBodyPosition).normalized;
+            
+            // TODO: fix this gravity force equation to make it more realistic to a black hole, and a greater force the closer to the singularity the object is
+            // var proximityMultiplier = pullCollider.radius - Vector3.Distance(singularityPosition, pulledBodyPosition);
+            pulledBody.AddForce(Time.fixedDeltaTime * gravitationalForce /* * proximityMultiplier */ * direction);
```

Comment:
Equation is `F = G * ( (M1 * M2) / (D)^2 )`
Essentially, just do 
```
var strength = (Time.fixedDeltaTime * gravitationalForce) / Mathf.Pow(Vector3.Distance(singularityPosition, pulledBodyPosition), 2);
pulledBody.AddForce(direction * strength);
```

Reply:
Why ignoring the masses?

Reply:
We can include them, but the numerator is essentially some gravitational force constant we can set. The distance between centers is what matters most for realism.

Reply:
Moved to this ticket: https://chilipot.atlassian.net/browse/PROP-33